Plugins.cpp

```diff
@@ -52,8 +52,9 @@ PluginInfo::PluginInfo() {
 }
 
 Plugins::Plugins(QObject *p) : QObject(p) {
-	qsSystemPlugins = QLatin1String("plugins/");
-	qsUserPlugins = QLatin1String("./");
+	QString AppDataLocation = QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
+	qsSystemPlugins = AppDataLocation + "/AppData/Roaming/Mumble/Plugins";
```

Comment:
The Mumble code for this is in Global.cpp 174ff. . For Qt 5 it uses ```QStandardPaths::writableLocation(QStandardPaths::DataLocation)``` and later only tacks on the Plugins directory. For that to return the correct folder you'll likely also have to set the application name to "Mumble" (see main.cpp of mumble 104ff.) as the path is application specific.